template/summary.html.tmpl

```diff
@@ -336,8 +337,9 @@
       </tr>
       {{ end }}
     </table>
-
-    <hr>
+  
+    <br />
```

Comment:
In HTML5 this tag is commonly spelled `<br>` (like `<hr>`).

In this case we could just use CSS instead.

Reply:
Yes, probably a better idea to use css.